x/incentive/client/tests/suite.go

```diff
@@ -0,0 +1,18 @@
+package tests
+
+import (
+	"testing"
+
+	"github.com/cosmos/cosmos-sdk/testutil/network"
+	itestsuite "github.com/umee-network/umee/v4/tests/cli"
+)
+
+type IntegrationTests struct {
+	*itestsuite.E2ESuite
+}
+
+func NewIntegrationTestSuite(cfg network.Config, t *testing.T) *IntegrationTests {
+	return &IntegrationTests{
+		&itestsuite.E2ESuite{Cfg: cfg, T: t},
+	}
+}
```

Comment:
this is not an integration test suite. It's an e2e test suite, and should be moved to `tests/e2e/incentive`

Reply:
Should we also move the matching `x/leverage/client/tests/suite.go` to e2e? It looks like the makefile won't need to be changed, so this seems doable.